content/blog/rajagopalan/2019-06-20_coding_week_no_2_and_3.md

```diff
@@ -0,0 +1,46 @@
++++
+type = "blog"
+title = "Coding week no 2 and 3"
+author = "rajagopalan"
+date = "2019-06-20 17:11:01+05:30"
+tags = ["haiku", "software","gosc2019","webkit"]
++++
+
+Love that your beautiful face is reading my Blog today ❤️.
+
+First all I would like to ask sorry to my fans (if any 😅😂) for delaying the posts. Prepare to get blown away🎉🎉💻
+
+Ok let's all jump into the world of webkit shall we?
+
+<h1> History is awesome (just kidding 😉😉)</h1>
+Lets rewind a bit!
+
+* Getting stuff to compile and build (painful but totally worth it)
+* Oh ya not to mention >:making friends!:<
+* Taming git and messing up with the repo.
+* Phew and finally getting the processes to get to know about each other (duh by making them talk).
+
+<h1>Living in the Present</h1>
+Ah yes after completing these I feel totally awesome and stepped up the game, And that brings us to the latest progress....
+
+Now the main feature of a browser is to show webpages. hell no!! It has got other cool stuff too.
+
+So first I did easy things like showing the status of the progress, keeping track of history( to go back and forward 🚗). For this I had to complete implementing Shared memory :heavenly tones play: . Sounds cool eh? Yes Shared memory is about allocating memory and sharing it with other processes ,this is done to keep track of visited processes.
+
+(Psst [click-here](https://imgur.com/a/0Ny2sj5) for some cool stuff)
```

Comment:
Please add the images to our git repo instead, we don't know if imgur is going to keep them long enough (we have some 15 years old articles where images went missing and it's annoying)

Reply:
Last time the image was just gone, maybe I followed wrong procedure?
Whats the correct one to add image?